spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -65,29 +67,12 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
             } else if (!testExists) {
               fail(s"Missing test: $expectedTestName")
             }
-          } else if (testExists) {
-            fail(s"Found test for cast that Spark does not support: $expectedTestName")
```

Comment:
Was there a particular test that was incorrectly failing due to this check?

Reply:
Not yet, but it would fail once we add tests for casting to/from TimestampNTZType because Spark 3.2 doesn't have that one. This change is unrelated to this PR though, so happy to remove it.

Reply:
Actually, the list of supported types wouldn't include TimestampNTZType when running against Spark 3.2 .. I will revert this change.